Makefile

```diff
@@ -23,12 +23,19 @@ c-lightning.s9pk: manifest.yaml assets/compat/* image.tar instructions.md $(ASSE
 	embassy-sdk pack
 
 image.tar: Dockerfile docker_entrypoint.sh check-rpc.sh check-synced.sh configurator/target/aarch64-unknown-linux-musl/release/configurator c-lightning-http-plugin/target/aarch64-unknown-linux-musl/release/c-lightning-http-plugin $(C_LIGHTNING_GIT_FILE) $(PLUGINS_SRC) $(C_LIGHTNING_REST_SRC) manifest.yaml
+# dumb hack to give the c-lightning build a .git folder because it insists on breaking on a submodule .git file
```

Comment:
:speech_balloon: Thanks for indicating that this was a hack. This was better than the last hack, so boyscout rule says ok. Can't get perfect, and can't think of a perfect. I'm good with this hack

Reply:
Sorry, the build still re-executes every time. I can't figure out how to make it use the docker cache.